build/Targets/RepoToolset/Imports.targets

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsShipping Condition="'$(IsShipping)' == ''">true</IsShipping>
```

Comment:
Why do we prefer shipping as the default state? I thought RepoToolset worked the other way: require opt-in for shipping, not opt-out.

Reply:
RepoToolset has the same default. The reason is that most projects that are not test projects are shipping. The only non-shipping non-test projects are utilities and helper tools.

Reply:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Imports.targets#L29

Reply:
I'm not sure I follow. Are you suggesting it should be conditional where it is set to false for tests?